src/material/slide-toggle/slide-toggle.spec.ts

```diff
@@ -759,6 +762,15 @@ describe('MatSlideToggle with forms', () => {
       expect(slideToggle.disabled).toBe(false);
       expect(inputElement.disabled).toBe(false);
     });
+
+    it('should not change focus origin if origin not specified', () => {
+      slideToggle.focus('mouse');
```

Comment:
Similar issue to here: https://github.com/angular/components/pull/20912.

Would like to call focus on the button in the form, setting the origin to some value and then calling focus on slideToggle and expecting it to remain unchanged.